ubuntu-deploy/index.md

```diff
@@ -0,0 +1,414 @@
+---
+title: 'Hosting a Directus Application on an Ubuntu Server'
+description: 'Learn how to host a Directus application on Ubuntu server with Docker, Nginx, and SSL encryption. A step-by-step guide for seamless deployment and security.'
+author:
+  name: 'Yusuf Akorede'
+  avatar_file_name: 'avatar.png'
+---
+
+## Introduction
+
+In this tutorial, you will learn how to deploy a Directus application within a Docker container on an Ubuntu Linux server and connect it to a custom domain, such as `https://directus.exampledomain.com`.
```

Comment:
```suggestion
In this tutorial, you will learn how to deploy a Directus application within a Docker container on an Ubuntu server and connect it to a custom domain. Ubuntu is a popular open source Linux distribution  which is commonly available from hosting providers. 
```

Adding a one-liner about what Ubuntu is just in case folks don't know.